index.html

```diff
@@ -0,0 +1,150 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale =1.0">
+        <title>Portfolio project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/d901e0cb9a.js" crossorigin="anonymous"></script>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+<style>
+body {
+    font-family: 'Poppins';
+}
+</style>
+    </head>
+    <body>
+      <nav>
+        <ul>
+          <li><a class="logolink" href="">My Logo</a></li>
+          <li style="float: right;"><a href=""><img src="hammenu.png" alt="ham" style="width: 45px;height: 45px;"></a></li>
+        </ul>
+      </nav>
```

Comment:
For semantic purposes, I recommend Your `<nav>` should be wrapped inside your `<header>` tag. this aids accessibility for screen readers🤗